test/tcs_test/tcsmolten_salt_cases.h

```diff
@@ -22,4 +25,287 @@ int tcsmolten_salt_daggett(ssc_data_t &data)
 	return status;
 }
 
+// Power Tower molten salt with alternative Rankine Cycle configuration
+// Rankine Power Cycle: Sliding pressure & Air-cooled
+// Rest default configurations
+int tcsmolten_salt_daggett_sliding_pressure(ssc_data_t &data)
+{
+	tcsmolten_salt_default(data);
+
+	ssc_data_set_number(data, "tech_type", 3);
+	
+	int status = run_module(data, "tcsmolten_salt");
+
+	//single_owner_default(data);
+	//status += run_module(data, "singleowner");
+
+	return status;
+}
+
+// Power tower molten salt with alternative Rankine Cycle configuration
+// Condenser type alternative: Evaporative 
+// Rest default configurations 
+int tcsmolten_salt_daggett_evap_condenser(ssc_data_t &data)
+{
+	tcsmolten_salt_default(data);
+	ssc_data_set_number(data, "CT", 1);
+	int status = run_module(data, "tcsmolten_salt");
+
+	//single_owner_default(data);
+	//status += run_module(data, "singleowner");
+
+	return status;
+}
+
+// Power tower molten salt with alternative Rankine Cycle configuration
+// Condenser type alternative: Hybrid
+// Rest default configurations 
+int tcsmolten_salt_daggett_hybrid_condenser(ssc_data_t &data)
+{
+	tcsmolten_salt_default(data);
+	ssc_data_set_number(data, "CT", 3);
+	int status = run_module(data, "tcsmolten_salt");
+
+	//single_owner_default(data);
+	//status += run_module(data, "singleowner");
+
+	return status;
+}
+
+// Power tower molten salt with alternative Rankine Cycle configuration
+// Condenser type alternative: Radiative
+// Rest default configurations 
+int tcsmolten_salt_daggett_radiative_condenser(ssc_data_t &data)
+{
+	tcsmolten_salt_default(data);
+	// Start of radiative-cooling metrics
+	ssc_data_set_number(data, "h_ctes_tank_min", 1);
+	ssc_data_set_number(data, "ctes_tshours", 15);
+	ssc_data_set_number(data, "ctes_field_fl", 4);
+	ssc_data_set_number(data, "h_ctes_tank", 30);
+	ssc_data_set_number(data, "u_ctes_tank", 0.40000000000000002);
```

Comment:
Identify when the generated code value is trying to approximate a less significant [digit] value. For this instance change to 0.4. Update all other similar instances and remember to ensure that the test(s) still pass.